hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -219,6 +219,58 @@
       <scope>test</scope>
     </dependency>
 
+    <!--
+    the mapreduce-client-core module is compiled against for the
+    manifest committer support. It is not needed on the classpath
+    except when using this committer, so it only tagged as
+    "provided".
+    -->
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-mapreduce-client-core</artifactId>
```

Comment:
This dependency should not be coming in. CloudStorageConnectors need to be usable without caring about libraries like mapreduce being pulled in, or being present on the classpath.